design/census-import/config/sql.config.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+require('dotenv').config()
+const toBool = require('to-bool')
+
+const oneMinuteInMilliseconds = 60000
+
+module.exports = {
+  database: process.env.SQL_DATABASE || 'mtc',
```

Comment:
Why do you need another config file?

Reply:
@jon-shipley I personally like having an isolated config for the sql. config js contains everything else as it stands for this small project.